CallCenterSample/Helpers/TextAnalyticsHelper.cs

```diff
@@ -109,48 +123,41 @@ public static async Task<DetectLanguageResult> GetDetectedLanguageAsync(string i
 
             if (!string.IsNullOrEmpty(input))
             {
-                LanguageBatchResult result = await AnalyticsClient.DetectLanguageAsync(new BatchInput(
-                    new List<Input>()
-                    {
-                        new Input("0", input)
-                    }));
-
-                if (result.Documents != null)
-                {
-                    languageResult.Language.Add("iso6391Name", result.Documents[0].DetectedLanguages[0].Iso6391Name);
-                    languageResult.Language.Add("name", result.Documents[0].DetectedLanguages[0].Name);
-                    languageResult.Language.Add("score", result.Documents[0].DetectedLanguages[0].Score.ToString());
-                }
+                DetectLanguageResultCollection result = await AnalyticsClient.DetectLanguageBatchAsync(
```

Comment:
Because there is only 1 document, it will be better to do `DetectedLanguage result = await AnalyticsClient.DetectLanguageAsync(input);`
See [sample](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample1_DetectLanguage.md#detecting-a-language-for-a-single-document) for more information

Reply:
https://github.com/Azure-Samples/cognitive-services-csharp-call-center/pull/2#discussion_r528560302